Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Models/LambdaMethodModel.cs

```diff
@@ -99,7 +114,7 @@ public bool ReturnsIHttpResults
         /// <summary>
         /// Gets or sets type of Lambda event
         /// </summary>
-        public List<EventType> Events { get; set; }
+        public HashSet<EventType> Events { get; set; }
```

Comment:
We only care about number of unique events and therefore the type is changed from `List` to `HashShet`